### PR TITLE
[top_englishbreakfast] Add second UART

### PR DESCRIPTION
**Pull request by vogelpi**
For penetration testing, we've been using a second UART on the CW310 and the silicon platform such that the host can directly communicate with the target using the uJSON protocol instead of looping this communication through the simpleserial channel with the scope (available on Husky only anyways).

This commit now adds a second UART to the English Breakfast top level such that we can re-align the capture setups again. Ultimately, this will allow removing the simpleserial-based communication which will quite notably simplify code maintenance in the future.

For reference, see also https://github.com/lowRISC/ot-sca/issues/235.

### CODE REVIEW

**Comment by vogelpi** [comment]
@nasahlpa , @andrea-caforio , I've successfully tested this using simpleserial on the FPGA connected to our shared workstation. To get the uJSON communication working in ot-sca CI, we'll also need to connect an additional FTDI to the IO5 and IO6 pins of the JP3 header. This is because on the CW305, the integrated of UARTs/USARTs of the SAM3U microcontroller are unfortunately not connected to the FPGA.

**Review by pamaury — APPROVED** [review]
The software side looks good to me

File: hw/top_englishbreakfast/data/top_englishbreakfast.hjson [hunk 1]
```diff
@@ -186,13 +186,28 @@
       // the ip.hjson will declare the clock port names
       // If none are defined at ip.hjson, clk_i is used by default
       clock_srcs: {clk_i: "io_div4"},
-
+      clock_group: "peri",
       // reset connections defines the port to top level reset connection
       // the ip.hjson will declare the reset port names
       // If none are defined at ip.hjson, rst_ni is used by default
-      reset_connections: {rst_ni: "sys_io_div4"},
+      reset_connections: {rst_ni: "lc_io_div4"},
```
Comment (nasahlpa) [inline hw/top_englishbreakfast/data/top_englishbreakfast.hjson:193]: Why are we now using `lc_io_div4` instead of `sys_io_div4`?

**Review by nasahlpa — APPROVED** [review]
LGTM, just one question.

File: hw/top_englishbreakfast/data/top_englishbreakfast.hjson (hunk 1 above)
Comment (vogelpi) [reply hw/top_englishbreakfast/data/top_englishbreakfast.hjson:193]: `rst_sys` resets the entire system where as `rst_lc` resets the system excluding rv_dm and a small portion of pinmux.  I noted that the same change was done a while ago for Earlgrey (but then we forgot to update English Breakfast in sync).

**Review by andrea-caforio — APPROVED** [review]

**Comment by vogelpi** [comment]
Thanks for your reviews guys!

**Pull request merged**